### PR TITLE
SharedId userid submodule: sync flag

### PR DESCRIPTION
**Pull request by yava-ua**
## Type of change
- [X] Feature
- [X] Code style update (formatting, local variables)
- [X] Does this change affect user-facing APIs or examples documented on http://prebid.org?

## Description of change
Prebid - SharedId sync flag. reuses param property syncTime. Anything less than 0 means sync is disabled.

### CODE REVIEW

File: modules/sharedIdSystem.js [hunk 1]
```diff
@@ -54,19 +55,23 @@ function isIdSynced(configParams, storedId) {
     utils.logInfo('SharedId: Sync time is not configured or is not a number');
   }
   let syncTime = (!configParams || typeof configParams.syncTime !== 'number') ? DEFAULT_24_HOURS : configParams.syncTime;
+  if (syncTime < 0) {
```
Comment (yava-ua) [inline modules/sharedIdSystem.js:58]: @jdwieland8282 reused syncTime parameter. Is this ok? Less than zero (e.g.`syncTime: -1`)  would mean sync is not required.

**Review by SKOCHERI — APPROVED** [review]

**Comment by patmmccann** [comment]
@nothingismatters @jdwieland8282 what is the status of this pr? Does it create a merge conflict with your other work?

File: modules/sharedIdSystem.js (hunk 1 above)
Comment (patmmccann) [reply modules/sharedIdSystem.js:58]: Should this be <= 0 to match other config?

File: modules/sharedIdSystem.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ The below parameters apply only to the Shared ID User ID Module integration.
 | --- | --- | --- | --- | --- |
 | name | Required | String | ID value for the Shared ID module - `"sharedId"` | `"sharedId"` |
 | params | Optional | Object | Details for sharedId syncing. | |
-| params.syncTime | Optional | Object | Configuration to define the frequency(in seconds) of id synchronization. By default id is synchronized every 24 hours | 60 |
+| params.syncTime | Optional | Object | Configuration to define the frequency(in seconds) of id synchronization. Less than 0 means periodic sync is disabled. By default id is synchronized every 24 hours | 60 |
```
Comment (patmmccann) [inline modules/sharedIdSystem.md:39]: Can you set <=

**Review by patmmccann — CHANGES_REQUESTED** [review]

**Comment by NothingIsMatter** [comment]
> @NothingIsMatter @jdwieland8282 what is the status of this pr? Does it create a merge conflict with your other work?

@patmmccann rather yes, than no

**Comment by patmmccann** [comment]
Sorry just to confirm what you are saying,  this pr does create a merge conflict and it should be closed?

**Comment by NothingIsMatter** [comment]
> Sorry just to confirm what you are saying, this pr does create a merge conflict and it should be closed?

@patmmccann wait, I missed that this PR is made against master branch, since my changes are made against prebid-5.0 branch i think they will not conflict

**Comment by patmmccann** [comment]
Okay @yana-ua could you just enable 0 as a value that disables and we can merge?

**Comment by jdwieland8282** [comment]
Sounds like this PR is creating a unnecessary distraction, and possibly a larger problem if master ever got merged into 5.0. Since the sharedid module is going away, we should stop development efforts against it.

**Comment by patmmccann** [comment]
Feel free to reopen after 5 in a few weeks.

**Pull request closed without merging**